pkg/cmd/service/service_test.go

```diff
@@ -208,32 +219,24 @@ type authTest struct {
 }
 
 func TestBuildServiceWithNoAuth(t *testing.T) {
-	config, err := GetServiceConfig()
-	require.NoError(t, err)
+	config := DefaultConfig()
```

Comment:
The reason we used `GetServiceConfig` was that it is the config entrypoint for the service. It is supposed to call `DefaultConfig` and merge other user provided configs ontop of it. 

I think that we actually get better test coverage and test fidelity if we leave this as it was before. It also gives us the apportunity to use `os.SetEnv(...)` to test that our env variables are wired up in our service properly and that `GetServiceConfig` respects it. So I think I'd prefer to leave it even though it is one extra line of code for the `require.NoError(...)`.

Reply:
It makes sense. I thought that if we need it in a particular test we could use it, but I see what you are saying too. Reverted. Thanks!